src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -747,6 +747,7 @@ public override void OnFocusLost()
         /// <param name="touch">The touch event.</param>
         /// <returns>True if the event should be consumed.</returns>
         /// <since_tizen> 8 </since_tizen>
+        [Obsolete("Deprecated in API8; Will be removed in API10.")]
```

Comment:
Are there any suggested methods that should be used instead of it? If so, please write it in the description of the ObsoleteAttribute.

Reply:
> Are there any suggested methods that should be used instead of it? If so, please write it in the description of the ObsoleteAttribute.

I changed it to:
[Obsolete("Deprecated in API8; Will be removed in API10. Please use OnClicked instead.")]